src/scraper/robot/util/find-page-count.ts

```diff
@@ -6,12 +6,9 @@ import { Logger } from "../../util/logger";
  * but rather requires to iterate through pages to find the last page.
  * @param fn A function which returns whether a given page has items
  */
-export async function findPageCount(fn: (page: number) => Promise<boolean>): Promise<number> {
+export async function findPageCount(fn: (page: number) => Promise<boolean>, min: number = 1, max: number = 100): Promise<number> {
```

Comment:
Please allow the user to simply call findPageCount(fn) or call findPageCount(config), where config is an object which contains fn, min and max, so that the code as simple as possible.